### DISCUSSION TITLE
Set Termux as lockscreen

### DISCUSSION THREAD

**Discussion by McTRASH692** [termux/termux-app#3124] [Ideas]
I'm sorry if this has been done and discussed but I was wondering if there was a way to display the current shell as an active lockscreen? I don't want it to be interactive but I do want it to update.

I want to use it on an old device to monitor my pihole over ssh using piholes chronometer.
But I have kids and they are terrors that like to get into things and I don't want them to be able to get into the device.

Is there an add-on or a way of doing this?

EDIT: Spelling.

**Comment by sylirre** [comment]
Unlikely to be worked on unless there are people who want to develop and maintain such add-on.

**Reply by McTRASH692** [reply]
I would be willing to work on such a project but would need a fair bit of guidance, my scripting ability is amature at best.

**Comment by trygveaa** [comment]
Most Android devices support [pinning an app](https://support.google.com/android/answer/9455138) so you can't switch to another app without unlocking the device. Sounds like this is what you want? Have you checked if your device supports it?

**Reply by McTRASH692** [reply]
I'll look into it, it may work for what I want but I will check it out, thank you,
